### DISCUSSION TITLE
Starcoin miner and startup not working

### DISCUSSION THREAD

**Discussion by JCJourney** [starcoinorg/starcoin#2559] [General]
Hello, 

I just found out about this coin and I am very interested in this project. As I was going through the starcoin miner installation process it would not let me download the miner because I needed to download the file sharing software, which is not allowed from my region. Is there a way I can still be apart of this? If there is can you please let me know. Thank you for your time and hope to hear from yall soon.

**Comment by jolestar** [comment]
Who won't let you download the miner?  what is the file link? I didn't understand the problem.

**Comment by JCJourney** [comment]
Hello sorry I was busy these past few days, but I went to this link:
https://starcoin-miner.gitbook.io/starcoin-miner-guide/mi-ni-xing-kuang-ji/usb-qu-dong-an-zhuang-shuo-ming
to go through the set up process and then it lead me to this link to download the miner: 
https://pan.baidu.com/s/1UA12dTpN4MahsIHgCKDmOQ#list/path=%2F
After it leads me there it says i need to download some file sharing software thats restricted to other countries. If I am doing something wrong please let me know. But I was just following the instructions given to download the miner.

**Reply by sanlee42** [reply]
> Hello sorry I was busy these past few days, but I went to this link:
> https://starcoin-miner.gitbook.io/starcoin-miner-guide/mi-ni-xing-kuang-ji/usb-qu-dong-an-zhuang-shuo-ming
> to go through the set up process and then it lead me to this link to download the miner:
> https://pan.baidu.com/s/1UA12dTpN4MahsIHgCKDmOQ#list/path=%2F
> After it leads me there it says i need to download some file sharing software thats restricted to other countries. If I am doing something wrong please let me know. But I was just following the instructions given to download the miner.

"https://pan.baidu.com/s/1UA12dTpN4MahsIHgCKDmOQ#list/path=%2F"   is link to [stm32 virtual com port driver](https://www.st.com/en/development-tools/stm32-utilities.html#overview).Since the miner is a usb device. You don't need to install it for most of the time.

**Reply by JCJourney** [reply]
So do i need to go to that link to download it or can I get it elsewhere?

**Comment by JCJourney** [comment]
okay i found the miner now. Is there a step by step tutorial of how to configure everything?

**Reply by sanlee42** [reply]
Once the miner device found by the software, you can go to the step 5 on https://starcoin-miner.gitbook.io/starcoin-miner-guide/mi-ni-xing-kuang-ji/wa-kuang-shuo-ming for seting the mining pool
